### PR TITLE
Rename all occurences of "accelerator" into "gpu"

### LINKED ISSUE #2172: As a researcher using Intel GPUs I would like the ability to set the LevelZero Performance Factor
**Is your feature request related to a problem? Please describe.**
Some workloads can benefit from external performance hints to Intel GPUs via the performance factor interface.  Currently GEOPM does not support this signal or control.
Feature [documentation](https://spec.oneapi.io/level-zero/latest/sysman/PROG.html#performance-factor)

**Describe the solution you'd like**
Expose a series of LEVELZERO::GPU_PERFORMANCE_FACTOR* and GPUCHIP_PERFORMANCE_FACTOR* signals at various domains (package/all, compute, memory) similar to the existing LevelZero utilization signals. 
Similarly a series of controls for these domains should be exposed.

**Describe alternatives you've considered**
Relying upon hardware balancing - may result in perf loss vs hand tune values
Using a single setting per device - may result in perf loss on some workloads

**Additional context**

### PR DESCRIPTION
**Pull request by Krebrov001**
- Relates to #2171 
- Fixes #2172 

Rename all occurences of "accelerator" into "gpu" within files in the `MANIFEST`.
Also rename the actual files themselves:

```
service/src/AcceleratorTopo.cpp
service/src/AcceleratorTopo.hpp
service/src/AcceleratorTopoNull.cpp
service/src/AcceleratorTopoNull.hpp
service/src/LevelZeroAcceleratorTopo.cpp
service/src/LevelZeroAcceleratorTopo.hpp
service/src/NVMLAcceleratorTopo.cpp
service/src/NVMLAcceleratorTopo.hpp
service/test/AcceleratorTopoNullTest.cpp
service/test/LevelZeroAcceleratorTopoTest.cpp
service/test/MockAcceleratorTopo.hpp
service/test/NVMLAcceleratorTopoTest.cpp
```

### CODE REVIEW

**Comment by cmcantalupo** [comment]
accelerator -> gpu
Accelerator -> GPU
ACCELERATOR -> GPU

**Comment by lhlawson** [comment]
- [ ] **Intel GPU testing is still required.** 
- [x] NVIDIA GPU spot testing on an HPE SX40 found no issues. 
> HPE-SX40 basic reads return as expected, along with valid errors as expected for domain_idx out of bound (reading a gpu that doesn't exist) and unsupported domains (gpu_chip for this system):
> ```
> [hpe-sx40:~]geopmread GPU_MEMORY_ACTIVITY gpu 0
> 1.727236320129163e-08
> [hpe-sx40:~]geopmread GPU_COMPUTE_ACTIVITY gpu 0
> 0
> [hpe-sx40:~]geopmread GPU_MEMORY_ACTIVITY gpu 0
> 5.820883204884875e-09
> [hpe-sx40:~]geopmread GPU_POWER gpu 0
> 46.175
> [hpe-sx40:~]geopmread GPU_POWER board 0
> 177.898
> [hpe-sx40:~]geopmread GPU_POWER gpu 15
> Error: cannot read signal: <geopm> Invalid argument: PlatformIOImp::read_signal(): domain_idx is out of range: at src/PlatformIO.cpp:482
> [hpe-sx40:~]geopmread GPU_POWER gpu 3
> 43.754
> [hpe-sx40:~]geopmwrite GPU_FREQUENCY_CONTROL board 0 1530000000
> [hpe-sx40:~]geopmread GPU_FREQUENCY_STATUS board 0
> 1530000000
> [hpe-sx40:~]geopmwrite GPU_FREQUENCY_CONTROL gpu 0 1530000000
> [hpe-sx40:~]geopmwrite GPU_FREQUENCY_CONTROL gpu 0 135000000
> [hpe-sx40:~]geopmread GPU_FREQUENCY_STATUS gpu 0
> 135000000
> [hpe-sx40:~]geopmread GPU_FREQUENCY_STATUS board 0
> 1181250000
> [hpe-sx40:~]geopmread GPU_FREQUENCY_STATUS gpu_chip 0
> Error: cannot read signal: <geopm> Invalid argument: PlatformIOImp::read_signal(): domain 10 is not valid for signal "GPU_FREQUENCY_STATUS": at src/PlatformIO.cpp:543
> [hpe-sx40:~]geopmwrite NVML::GPU_FREQUENCY_RESET_CONTROL board 0 1
> [hpe-sx40:~]geopmread GPU_FREQUENCY_STATUS board 0
> 135000000
> ```
> 
> Testing with a basic PARRES dgemm run shows reports working:
> ```
>     Application Totals:
>       runtime (s): 15.2876
> <removed non-gpu readings as uninteresting to debug>
>       gpu-energy (J): 12835.6
>       gpu-power (W): 840.296
>       gpu-frequency (Hz): 1.43566e+09
> ```
> 
> Similarly tracing of GPU workloads appears to be unaffected by the change, beyond signal domain changes:
> image
> 
> Not tested for V100s: 
> - non-root privileges 
> - all GPUs, beyond testing of board level readings
> - accuracy of readings, beyond visual inspection vs previous readings on the same system.

**Comment by Krebrov001** [comment]
Please review
@cmcantalupo @bgeltz  @lhlawson

File: service/docs/source/security.rst [hunk 1]
```diff
@@ -283,7 +283,7 @@ LevelZero sysman library interface
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 The LevelZero sysman library interface enables users to monitor and
-control Intel accelerator devices. These signals and controls include
+control Intel gpu devices. These signals and controls include
```
Comment (bgeltz) [inline service/docs/source/security.rst]:
```suggestion
control Intel GPU devices. These signals and controls include
```

File: service/configure.ac [hunk 2]
```diff
@@ -97,7 +97,7 @@ AC_SUBST([enable_levelzero])
 AM_CONDITIONAL([ENABLE_LEVELZERO], [test "x$enable_levelzero" = "x1"])
 
 AC_ARG_ENABLE([nvml],
-  [AS_HELP_STRING([--enable-nvml], [Enables use of the NVML library to support NVML board accelerators])],
+  [AS_HELP_STRING([--enable-nvml], [Enables use of the NVML library to support NVML board gpus])],
```
Comment (bgeltz) [inline service/configure.ac]:
```suggestion
  [AS_HELP_STRING([--enable-nvml], [Enables use of the NVML library to support NVML board GPUs])],
```

File: service/configure.ac [hunk 3]
```diff
@@ -110,7 +110,7 @@ AC_SUBST([enable_nvml])
 AM_CONDITIONAL([ENABLE_NVML], [test "x$enable_nvml" = "x1"])
 
 AC_ARG_ENABLE([dcgm],
-  [AS_HELP_STRING([--enable-dcgm], [Enables use of the DCGM library to support NVIDIA board accelerators])],
+  [AS_HELP_STRING([--enable-dcgm], [Enables use of the DCGM library to support NVIDIA board gpus])],
```
Comment (bgeltz) [inline service/configure.ac]:
```suggestion
  [AS_HELP_STRING([--enable-dcgm], [Enables use of the DCGM library to support NVIDIA board GPUs])],
```

File: service/configure.ac [hunk 4]
```diff
@@ -150,17 +150,17 @@ AC_SUBST([enable_coverage])
 
 if test "x$enable_dcgm" = "x1" ; then
   ENABLE_DCGM=True
-  AC_DEFINE([GEOPM_ENABLE_DCGM], [ ], [Enables use of the DCGM library to support NVIDA board accelerators])
+  AC_DEFINE([GEOPM_ENABLE_DCGM], [ ], [Enables use of the DCGM library to support NVIDA board gpus])
```
Comment (bgeltz) [inline service/configure.ac]:
```suggestion
  AC_DEFINE([GEOPM_ENABLE_DCGM], [ ], [Enables use of the DCGM library to support NVIDA board GPUs])
```

File: service/configure.ac [hunk 5]
```diff
@@ -150,17 +150,17 @@ AC_SUBST([enable_coverage])
...
 fi
 
 if test "x$enable_nvml" = "x1" ; then
   ENABLE_NVML=True
-  AC_DEFINE([GEOPM_ENABLE_NVML], [ ], [Enables use of the NVML library to support NVML board accelerators])
+  AC_DEFINE([GEOPM_ENABLE_NVML], [ ], [Enables use of the NVML library to support NVML board gpus])
```
Comment (bgeltz) [inline service/configure.ac]:
```suggestion
  AC_DEFINE([GEOPM_ENABLE_NVML], [ ], [Enables use of the NVML library to support NVML board GPUs])
```

File: service/configure.ac [hunk 6]
```diff
@@ -150,17 +150,17 @@ AC_SUBST([enable_coverage])
...
 else
   ENABLE_NVML=False
     if test "x$enable_levelzero" = "x1" ; then
         ENABLE_LEVELZERO=True
-        AC_DEFINE([GEOPM_ENABLE_LEVELZERO], [ ], [Enables use of the levelzero library to support levelzero board accelerators])
+        AC_DEFINE([GEOPM_ENABLE_LEVELZERO], [ ], [Enables use of the levelzero library to support levelzero board gpus])
```
Comment (bgeltz) [inline service/configure.ac]:
```suggestion
        AC_DEFINE([GEOPM_ENABLE_LEVELZERO], [ ], [Enables use of the levelzero library to support levelzero board GPUs])
```

File: service/src/DCGMDevicePool.hpp [hunk 7]
```diff
@@ -38,29 +38,29 @@ namespace geopm
             DCGMDevicePool() = default;
             virtual ~DCGMDevicePool() = default;
 
-            /// @brief Number of accelerators that support DCGM on the platform.
-            /// @return Number of accelerators supported by DCGM.
+            /// @brief Number of gpus that support DCGM on the platform.
```
Comment (bgeltz) [inline service/src/DCGMDevicePool.hpp]:
```suggestion
            /// @brief Number of GPUs that support DCGM on the platform.
```

File: service/src/DCGMDevicePool.hpp [hunk 8]
```diff
@@ -38,29 +38,29 @@ namespace geopm
...
             virtual int num_device() const = 0;
             /// @brief Get the value for the provided geopm_field_id.
             ///
             /// This value should not change unless update_field_value
             /// has been called
             ///
-            /// @param [in] accel_idx The index indicating a particular
-            ///        accelerator.
+            /// @param [in] gpu_idx The index indicating a particular
+            ///        gpu.
```
Comment (bgeltz) [inline service/src/DCGMDevicePool.hpp]:
```suggestion
            ///        GPU.
```

File: service/src/DCGMDevicePool.hpp [hunk 9]
```diff
@@ -38,29 +38,29 @@ namespace geopm
...
             ///
             /// @param [in] field_id One of the m_field_id_e values
             ///
             /// @return The value for the specified field
-            virtual double sample(int accel_idx, int field_id) const = 0;
+            virtual double sample(int gpu_idx, int field_id) const = 0;
 
-            /// @brief Query DCGM for the latest value for an accelerator.
+            /// @brief Query DCGM for the latest value for an gpu.
```
Comment (bgeltz) [inline service/src/DCGMDevicePool.hpp]:
```suggestion
            /// @brief Query DCGM for the latest value for an GPU.
```

File: service/src/DCGMDevicePool.hpp [hunk 10]
```diff
@@ -38,29 +38,29 @@ namespace geopm
...
             ///        Note that this is the last value DCGM cached.  This
             ///        updates the DCGM device pool stored value that is provided
             ///        via the sample_field_value function
-            /// @param [in] accel_idx The index indicating a particular
-            ///        accelerator.
-            virtual void update(int accel_idx) = 0;
+            /// @param [in] gpu_idx The index indicating a particular
+            ///        gpu.
```
Comment (bgeltz) [inline service/src/DCGMDevicePool.hpp]:
```suggestion
            ///        GPU.
```

File: service/src/GPUTopo.hpp [hunk 11]
```diff
@@ -3,32 +3,32 @@
...
-    class AcceleratorTopo
+    class GPUTopo
     {
         public:
-            AcceleratorTopo() = default;
-            virtual ~AcceleratorTopo() = default;
-            /// @brief Number of accelerators on the platform.
+            GPUTopo() = default;
+            virtual ~GPUTopo() = default;
+            /// @brief Number of gpus on the platform.
```
Comment (bgeltz) [inline service/src/GPUTopo.hpp]:
```suggestion
            /// @brief Number of GPUs on the platform.
```

File: service/src/GPUTopo.hpp [hunk 12]
```diff
@@ -3,32 +3,32 @@
...
             /// @param [in] domain The GEOPM domain type
-            virtual int num_accelerator(void) const = 0;
-            virtual int num_accelerator(int domain) const = 0;
-            /// @brief CPU Affinitization set for a particular accelerator
+            virtual int num_gpu(void) const = 0;
+            virtual int num_gpu(int domain) const = 0;
+            /// @brief CPU Affinitization set for a particular gpu
```
Comment (bgeltz) [inline service/src/GPUTopo.hpp]:
```suggestion
            /// @brief CPU Affinitization set for a particular GPU
```

File: service/src/GPUTopo.hpp [hunk 13]
```diff
@@ -3,32 +3,32 @@
...
             /// @param [in] domain The GEOPM domain type
             /// @param [in] domain_idx The index indicating a particular
-            ///        accelerator
+            ///        gpu
```
Comment (bgeltz) [inline service/src/GPUTopo.hpp]:
```suggestion
            ///        GPU
```

File: service/src/LevelZero.cpp [hunk 14]
```diff
@@ -350,24 +350,24 @@ namespace geopm
...
     }
 
-    int LevelZeroImp::num_accelerator() const
+    int LevelZeroImp::num_gpu() const
     {
-        //  TODO: this should be expanded to return all supported accel types.
-        //  Right now that is only board_gpus
-        return num_accelerator(GEOPM_DOMAIN_BOARD_ACCELERATOR);
+        //  TODO: this should be expanded to return all supported gpu types.
+        //  Right now that is only gpus
```
Comment (bgeltz) [inline service/src/LevelZero.cpp]:
```suggestion
        //  TODO: this should be expanded to return all supported GPU types.
        //  Right now that is only GPUs
```

File: service/src/LevelZero.cpp [hunk 15]
```diff
@@ -350,24 +350,24 @@ namespace geopm
...
+        return num_gpu(GEOPM_DOMAIN_GPU);
     }
 
-    int LevelZeroImp::num_accelerator(int domain_type) const
+    int LevelZeroImp::num_gpu(int domain_type) const
     {
-        //  TODO: this should be expanded to return all supported accel types.
-        //  Right now that is only board_gpus
+        //  TODO: this should be expanded to return all supported gpu types.
+        //  Right now that is only gpus
```
Comment (bgeltz) [inline service/src/LevelZero.cpp]:
```suggestion
        //  TODO: this should be expanded to return all supported GPU types.
        //  Right now that is only GPUs
```

File: service/src/LevelZero.hpp [hunk 16]
```diff
@@ -26,129 +26,129 @@ namespace geopm
 
             LevelZero() = default;
             virtual ~LevelZero() = default;
-            /// @brief Number of accelerators on the platform.
-            /// @return Number of LevelZero accelerators.
-            virtual int num_accelerator() const = 0;
+            /// @brief Number of gpus on the platform.
+            /// @return Number of LevelZero gpus.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            /// @brief Number of GPUs on the platform.
            /// @return Number of LevelZero GPUs.
```

File: service/src/LevelZero.hpp [hunk 17]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
+            virtual int num_gpu() const = 0;
 
-            /// @brief Number of accelerators on the platform.
+            /// @brief Number of gpus on the platform.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            /// @brief Number of GPUs on the platform.
```

File: service/src/LevelZero.hpp [hunk 18]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
             /// @param [in] domain The GEOPM domain type being targeted
-            /// @return Number of LevelZero accelerators or accelerator chips.
-            virtual int num_accelerator(int domain) const = 0;
+            /// @return Number of LevelZero gpus or gpu chips.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            /// @return Number of LevelZero GPUs or GPU chips.
```

File: service/src/LevelZero.hpp [hunk 19]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
             /// @brief Get the number of LevelZero frequency domains of a certain type
             /// @param [in] l0_domain The LevelZero domain type being targeted
-            /// @return Accelerator frequency domain count.
+            /// @return GPU frequency domain count.
             virtual int frequency_domain_count(unsigned int l0_device_idx,
                                                int l0_domain) const = 0;
             /// @brief Get the LevelZero device actual frequency in MHz
             /// @param [in] l0_device_idx The index indicating a particular
-            ///        Level Zero accelerator.
+            ///        Level Zero gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        Level Zero GPU.
```

File: service/src/LevelZero.hpp [hunk 20]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
             /// @param [in] l0_domain The LevelZero domain type being targeted
             /// @param [in] l0_domain_idx The LevelZero index indicating a particular
-            ///        domain of the accelerator.
-            /// @return Accelerator device core clock rate in MHz.
+            ///        domain of the gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        domain of the GPU.
```

File: service/src/LevelZero.hpp [hunk 21]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
+            /// @return GPU device core clock rate in MHz.
             virtual double frequency_status(unsigned int l0_device_idx,
                                             int l0_domain, int l0_domain_idx) const = 0;
             /// @brief Get the LevelZero device mininmum frequency in MHz
             /// @param [in] l0_device_idx The index indicating a particular
-            ///        Level Zero accelerator.
+            ///        Level Zero gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        Level Zero GPU.
```

File: service/src/LevelZero.hpp [hunk 22]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
             /// @param [in] l0_domain The LevelZero domain type being targeted
             /// @param [in] l0_domain_idx The LevelZero index indicating a particular
-            ///        domain of the accelerator.
-            /// @return Accelerator minimum frequency in MHz.
+            ///        domain of the gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        domain of the GPU.
```

File: service/src/LevelZero.hpp [hunk 23]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
+            /// @return GPU minimum frequency in MHz.
             virtual double frequency_min(unsigned int l0_device_idx, int l0_domain,
                                          int l0_domain_idx) const = 0;
             /// @brief Get the LevelZero device maximum frequency in MHz
             /// @param [in] l0_device_idx The index indicating a particular
-            ///        Level Zero accelerator.
+            ///        Level Zero gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        Level Zero GPU.
```

File: service/src/LevelZero.hpp [hunk 24]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
             /// @param [in] l0_domain The LevelZero domain type being targeted
             /// @param [in] l0_domain_idx The LevelZero index indicating a particular
-            ///        domain of the accelerator.
-            /// @return Accelerator maximum frequency in MHz.
+            ///        domain of the gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        domain of the GPU.
```

File: service/src/LevelZero.hpp [hunk 25]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
+            /// @return GPU maximum frequency in MHz.
             virtual double frequency_max(unsigned int l0_device_idx, int l0_domain,
                                          int l0_domain_idx) const = 0;
             /// @brief Get the LevelZero device mininum and maximum frequency
             ///        control range in MHz
             /// @param [in] l0_device_idx The index indicating a particular
-            ///        Level Zero accelerator.
+            ///        Level Zero gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        Level Zero GPU.
```

File: service/src/LevelZero.hpp [hunk 26]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
             /// @param [in] l0_domain The LevelZero domain type being targeted
             /// @param [in] l0_domain_idx The LevelZero index indicating a particular
-            ///        domain of the accelerator.
-            /// @return Accelerator minimum and maximum frequency range in MHz.
+            ///        domain of the gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        domain of the GPU.
```

File: service/src/LevelZero.hpp [hunk 27]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
 
             /// @brief Get the number of LevelZero engine domains
             /// @param [in] l0_domain The LevelZero domain type being targeted
-            /// @return Accelerator engine domain count.
+            /// @return GPU engine domain count.
             virtual int engine_domain_count(unsigned int l0_device_idx, int l0_domain) const = 0;
             /// @brief Get the LevelZero device active time and timestamp in microseconds
             /// @param [in] l0_device_idx The index indicating a particular
-            ///        Level Zero accelerator.
+            ///        Level Zero gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        Level Zero GPU.
```

File: service/src/LevelZero.hpp [hunk 28]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
-            /// @return Accelerator active time and timestamp in microseconds.
+            ///        domain of the gpu.
+            /// @return GPU active time and timestamp in microseconds.
             virtual std::pair<uint64_t, uint64_t> active_time_pair(unsigned int l0_device_idx,
                                                                    int l0_domain,
                                                                    int l0_domain_idx) const = 0;
             /// @brief Get the LevelZero device active time in microseconds
             /// @param [in] l0_device_idx The index indicating a particular
-            ///        Level Zero accelerator.
+            ///        Level Zero gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        Level Zero GPU.
```

File: service/src/LevelZero.hpp [hunk 29]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
             /// @param [in] l0_domain The LevelZero domain type being targeted
             /// @param [in] l0_domain_idx The LevelZero index indicating a particular
-            ///        domain of the accelerator.
-            /// @return Accelerator active time in microseconds.
+            ///        domain of the gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        domain of the GPU.
```

File: service/src/LevelZero.hpp [hunk 30]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
+            /// @return GPU active time in microseconds.
             virtual uint64_t active_time(unsigned int l0_device_idx, int l0_domain,
                                          int l0_domain_idx) const = 0;
             /// @brief Get the cachced LevelZero device timestamp for the
             ///        active time value in microseconds
             /// @param [in] l0_device_idx The index indicating a particular
-            ///        Level Zero accelerator.
+            ///        Level Zero gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        Level Zero GPU.
```

File: service/src/LevelZero.hpp [hunk 31]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
             /// @param [in] l0_domain The LevelZero domain type being targeted
             /// @param [in] l0_domain_idx The LevelZero index indicating a particular
-            ///        domain of the accelerator.
-            /// @return Accelerator device timestamp for the active time value in microseconds.
+            ///        domain of the gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        domain of the GPU.
```

File: service/src/LevelZero.hpp [hunk 32]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
+            /// @return GPU device timestamp for the active time value in microseconds.
             virtual uint64_t active_time_timestamp(unsigned int l0_device_idx,
                                                    int l0_domain, int l0_domain_idx) const = 0;
 
             /// @brief Get the LevelZero device default power limit in milliwatts
             /// @param [in] l0_device_idx The index indicating a particular
-            ///        Level Zero accelerator.
-            /// @return Accelerator default power limit in milliwatts
+            ///        Level Zero gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        Level Zero GPU.
```

File: service/src/LevelZero.hpp [hunk 33]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
+            /// @return GPU default power limit in milliwatts
             virtual int32_t power_limit_tdp(unsigned int l0_device_idx) const = 0;
             /// @brief Get the LevelZero device minimum power limit in milliwatts
             /// @param [in] l0_device_idx The index indicating a particular
-            ///        Level Zero accelerator.
-            /// @return Accelerator minimum power limit in milliwatts
+            ///        Level Zero gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        Level Zero GPU.
```

File: service/src/LevelZero.hpp [hunk 34]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
+            /// @return GPU minimum power limit in milliwatts
             virtual int32_t power_limit_min(unsigned int l0_device_idx) const = 0;
             /// @brief Get the LevelZero device maximum power limit in milliwatts
             /// @param [in] l0_device_idx The index indicating a particular
-            ///        Level Zero accelerator.
-            /// @return Accelerator maximum power limit in milliwatts
+            ///        Level Zero gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        Level Zero GPU.
```

File: service/src/LevelZero.hpp [hunk 35]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
+            /// @return GPU maximum power limit in milliwatts
             virtual int32_t power_limit_max(unsigned int l0_device_idx) const = 0;
 
             /// @brief Get the LevelZero device energy and timestamp
             ///        in microjoules and microseconds
             /// @param [in] l0_device_idx The index indicating a particular
-            ///        Level Zero accelerator.
-            /// @return Accelerator energy in microjoules and timestamp in microseconds
+            ///        Level Zero gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        Level Zero GPU.
```

File: service/src/LevelZero.hpp [hunk 36]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
+            /// @return GPU energy in microjoules and timestamp in microseconds
             virtual std::pair<uint64_t, uint64_t> energy_pair(unsigned int l0_device_idx) const = 0;
             /// @brief Get the LevelZero device energy in microjoules.
             /// @param [in] l0_device_idx The index indicating a particular
-            ///        Level Zero accelerator.
-            /// @return Accelerator energy in microjoules.
+            ///        Level Zero gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        Level Zero GPU.
```

File: service/src/LevelZero.hpp [hunk 37]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
+            /// @return GPU energy in microjoules.
             virtual uint64_t energy(unsigned int l0_device_idx) const = 0;
             /// @brief Get the LevelZero device energy cached timestamp in microseconds
             /// @param [in] l0_device_idx The index indicating a particular
-            ///        Level Zero accelerator.
-            /// @return Accelerator energy timestamp in microseconds
+            ///        Level Zero gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        Level Zero GPU.
```

File: service/src/LevelZero.hpp [hunk 38]
```diff
@@ -26,129 +26,129 @@ namespace geopm
...
+            /// @return GPU energy timestamp in microseconds
             virtual uint64_t energy_timestamp(unsigned int l0_device_idx) const = 0;
 
             /// @brief Set min and max frequency for LevelZero device.
             /// @param [in] l0_device_idx The index indicating a particular
-            ///        Level Zero accelerator.
+            ///        Level Zero gpu.
```
Comment (bgeltz) [inline service/src/LevelZero.hpp]:
```suggestion
            ///        Level Zero GPU.
```

**Comment by Krebrov001** [comment]
I removed the `rename_accelerator.py` from this branch/pull request `issue-2171`.

It is now located on the branch `rename_script`.

**Review by bgeltz — CHANGES_REQUESTED** [review]
We should be consistent about when GPU is capitalized in comments and error strings.  When it's used in a sentence or similar, it should always be GPU or GPUs.

I marked many of these, but I did not mark them all.  There are many more that need to be corrected.

**Review by bgeltz — APPROVED** [review]

**Review by cmcantalupo — APPROVED** [review]

**Pull request merged**